react-todo-app/src/components/TodoContainer.js

```diff
@@ -1,12 +1,107 @@
-import React from "react"
-class TodoContainer extends React.Component {
-  render() {
-    return (
-      <div>
-        <h1>Hello from Create React App</h1>
-        <p>I am in a React Component!</p>
-      </div>
+import React, { useState, useEffect } from "react"
+import { Route, Switch } from "react-router-dom"
```

Comment:
- There is an error here, because In react-router-dom v6, `Switch` is replaced by routes `Routes`. 
You need to update the import from 
``` import { Route, Switch } from "react-router-dom"; ```
to 
``` import { Route, Routes } from 'react-router-dom'; ```